s3po/backends/memory.py

```diff
@@ -23,3 +23,13 @@ def upload(self, bucket, key, fobj, retries, headers=None):
             self.buckets[bucket] = {}
 
         self.buckets[bucket][key] = fobj.read()
+
+    def list(self, bucket, prefix, delimiter, headers=None):
```

Comment:
Presumably this should be `prefix=None, delimiter=None` as well to match the others.